RUFAS/routines/animal/animal_module_reporter.py

```diff
@@ -404,6 +404,18 @@ def report_life_cycle_manager_data(life_cycle_manager, sim_day: int) -> None:
         )
         om.add_variable("cull_reason_stats", life_cycle_manager.cull_reason_stats, info_map)
 
+    def report_daily_pen_total(penlist):
+        info_map = {
+            "class": "AnimalModuleReporter",
+            "function": "report_daily_pen_total",
```

Comment:
```suggestion
            "class": AnimalModuleReporter.__class__.__name__,
            "function": AnimalModuleReporter.report_daily_pen_total.__name__,
```
having the class and function specified like this will help to avoid the situation where the class or method name changes but the info map doesn't.